text/001x-battery_improvements.md

```diff
@@ -0,0 +1,292 @@
+  * Start date: 2022-03-26
+  * Contributors: Hamish Willee <hamishwillee@gmail.com>, ...
+  * Related issues: 
+    - [16-cell support #1747](https://github.com/mavlink/mavlink/pull/1747)
+	- [common: added voltage and current multipliers to BATTERY_STATUS #233](https://github.com/ArduPilot/mavlink/pull/233)
+    - [Add MAV_BATTERY_CHARGE_STATE_CHARGING #1068](https://github.com/mavlink/mavlink/pull/1068)
+    - dronecan / DSDL : [WIP smart battery messages](https://github.com/dronecan/DSDL/pull/7) 
+
```

Comment:
@hamishwillee another valid document to consider for this RFC is the DS-013 standard 
https://docs.google.com/document/d/1xYBhhgjND_RS2W3Hq0Seyc-R661ge3zonk9DTIyHclc/edit

Reply:
```suggestion
- [DS-013 Pixhawk Smart Battery Standard](https://docs.google.com/document/d/1xYBhhgjND_RS2W3Hq0Seyc-R661ge3zonk9DTIyHclc/edit) v0.4.0

```

Reply:
I'd say the value of the extra temperatures vs. the link cost considering mavlink's main use is wireless isn't worth it. (Unlike in CAN)

But they do have a lot of value for higher end systems or those needing to fly in cold environments keeping track of how well their heater is coping.